src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -3307,7 +3304,7 @@ public Builder setWeeklyAnchor(WeeklyAnchor weeklyAnchor) {
           @Getter(onMethod_ = {@Override})
           private final String value;
 
-          DelayDays(String value) {
+          private DelayDays(String value) {
```

Comment:
This I think is the only remaining concern.

Reply:
Can you update the generator to remove the modifier so that the constructor stays at the package-private level?